README.md

```diff
@@ -185,7 +182,7 @@ spec:
 #### SLO
 
 A service level objective (SLO) is a target value or a range of values for
```

Comment:
If you are making change for `Service Level Indicator (SLI)` to be written uppercase. You should also change `service level objective(SLO)` to `Service Level Objective(SLO)` it is the same case next to each other.

Reply:
I'm making it consistent - lowercase everywhere as in the literature e.g. https://sre.google/sre-book/service-level-objectives/